apps/console-lite/src/app/components/simple-market-list/simple-market-list.tsx

```diff
@@ -129,6 +129,10 @@ const SimpleMarketList = () => {
     [handleRowClicked]
   );
 
+  const shouldSuppressHorizontalScroll = useMemo(() => {
+    return !isMobile && screenSize in { xl: '', xxl: '' };
```

Comment:
nitpick (non-blocking): This makes me feel strange. Not sure if it's because I haven't seen it before or if it is strange. Might be better 
`['xl', 'xxl'].includes(screenSize)`

Reply:
you're right, it's quite ugly :-( . Changed now